src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -274,6 +274,10 @@ Adds an integration to your configuration. Read more in [the integrations guide]
 
 Launches the Astro Docs website directly from the terminal.
 
+## `astro info`
```

Comment:
I thought that the commands were in alphabetic order.

Reply:
Apparently not :joy: